src/Core/Domain/Profile/ValueObject/ProfileId.php

```diff
@@ -47,7 +47,7 @@ public function __construct($profileId)
     {
         // Strict type should be used in next major
         if (!is_int($profileId)) {
-            @trigger_error('Invalid type, int is expected', E_STRICT);
+            @trigger_error('Invalid type, int is expected', E_USER_NOTICE);
```

Comment:
Why not `E_STRICT` any more?

Reply:
From https://www.php.net/manual/en/function.trigger-error.php:
>The designated error type for this error. It only works with the E_USER family of constants, and will default to E_USER_NOTICE